msteams-platform/concepts/build-and-test/deep-links.md

```diff
@@ -428,7 +429,83 @@ else { /* handle case where capability isn't supported */ }
 
 ```
 
-#### Generate a deep link to a call
+## Generate a deep link to share content to stage in meetings
+
+You can also generate a deep link to [share the app to stage](~/apps-in-teams-meetings/enable-and-configure-your-app-for-teams-meetings.md#share-entire-app-to-stage) and start or join a meeting.
+
+> [!Note]
+>
+> * Currently, deep link to share content to stage in meetings is available only in [public developer preview](~/resources/dev-preview/developer-preview-intro.md).
+> * Deep link to share content to stage in meeting is supported in Teams desktop client only.
+
+When a deep link is selected in an app by a user who is part of an ongoing meeting, then the app is shared to the stage and a permission pop-up window appears. Users can grant permissions for the participants, such as co-editing a document or collaborating with an app.
+
+> [!NOTE]
+> Permission pop-up window appears only for Office apps.
+
+:::image type="content" source="../../assets/images/intergrate-with-teams/screenshot-of-pop-up-permission.png" alt-text="The screenshot is an example that shows a permission pop-up window.":::
+
+When user isn't in a meeting then user is redirected to the Teams calendar where user needs to join a meeting or an instant meeting (Meet now) can be initiated.
+
+:::image type="content" source="../../assets/images/intergrate-with-teams/Instant-meetnow-pop-up.png" alt-text="The screenshot is an example that shows a pop-up window when there's no ongoing meeting.":::
+
+Once the user initiates an instant meeting (Meet now), they can add participants and interact with the app.
+
+:::image type="content" source="../../assets/images/intergrate-with-teams/Screenshot-ofmeet-now-option-pop-up.png" alt-text="The screenshot is an example that shows an option to add participants and how to interact with the app.":::
+
+To add a deep link to share content on stage, you need to have an app context. The app context allows the Teams client to fetch the app manifest and check if the sharing on stage is possible. The following is an example of an app context.
+
+* `{ "appSharingUrl" : "https://teams.microsoft.com/extensibility-apps/meetingapis/view", "appId": "9ec80a73-1d41-4bcb-8190-4b9eA9e29fbb" , “useMeetNow”: false }`
+
+The query parameters for the app context are:
+
+* `appID`: This is the ID that can be obtained from the app manifest.
```

Comment:
Can you please confirm that the manifest ID works even for LOB and sideloaded apps? This is a BIG source of bugs when this doesn't work.

Reply:
yes it will work for sideloaded scenario as well

Reply:
Thanks for confirming the sideloaded case. Will the manifest ID work for LOB apps too?

What does it mean that we're rolling that out separately? When this deeplink is released, will using the app ID that's in the manifest work in all cases, including LOB and sideloaded?

Reply:
sideloaded case was not present as a mvp feature initially but we were able to make changes to make it work
The changes are in Ring 3 already and are expected to be released in next few weeks

Reply:
Hello! I'm a developer at Figma working on implementing deep-linking. Just confirming that the `appID` parameter comes from the `id` of the manifest like so:

```
{
    "$schema": "https://raw.githubusercontent.com/OfficeDev/microsoft-teams-app-schema/preview/DevPreview/MicrosoftTeams.schema.json",
    "manifestVersion": "devPreview",
    "version": "1.0.0",
    "id": "{{appId}}",
    "packageName": "com.microsoft.teams.extension",
    ...
}
```

And also confirming if this works for side-loaded apps beyond Ring 3 (GA ring)?

Reply:
Hello! I'm a developer at Figma working on implementing deep-linking. Just confirming that the `appID` parameter comes from the `id` of the manifest like so:

```
{
    "$schema": "https://raw.githubusercontent.com/OfficeDev/microsoft-teams-app-schema/preview/DevPreview/MicrosoftTeams.schema.json",
    "manifestVersion": "devPreview",
    "version": "1.0.0",
    "id": "{{appId}}",
    "packageName": "com.microsoft.teams.extension",
    ...
}
```

And also confirming if this works for side-loaded apps beyond Ring 3 (GA ring)?

It's pretty cool to see some progress of it working locally:

<img width="1374" alt="Screen Shot 2022-10-06 at 3 30 23 PM" src="https://user-images.githubusercontent.com/12423542/194430421-943634bd-b088-43c5-8e94-dd50458777eb.png">

<img width="1003" alt="Screen Shot 2022-10-06 at 3 30 40 PM" src="https://user-images.githubusercontent.com/12423542/194430444-ac6a761b-c581-4fd0-a601-db7a1e0eed0e.png">

And wanted to ask about the failure states for the "URL is invalid" error message. I'm using the same app sharing URL that I use to use the Teams SDK to share to stage, so I know that in fact my application does have permissions to share this URL to stage. My theory is that it's the `appId` of my side-loaded app that isn't recognizable. Any pointers would be great!